main.go

```diff
@@ -293,6 +296,19 @@ func runCLIApp(c *cli.Context) (err error) {
 				p)
 		}
 
+		// Pass the parent process working directory to child process via
+		// environment variable. This variable will be used to resolve relative paths.
+		var parentProcessExecutionDir string
```

Comment:
we dont need variable definition, you can directly use parentProcessExecutionDir :=

Reply:
done.